src/Dropdown/stories/index.stories.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import { useState } from '@storybook/addons';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+
+import { UserCircle } from 'phosphor-react';
+import { default as Menu, MenuItem, MenuSubMenu, MenuItemGroup,MenuDivider } from '../../Menu';
+import { default as Dropdown, DropdownProps } from '../index'
+
+export default {
+  title: 'Dropdown',
+  component: Dropdown,
+} as ComponentMeta<typeof Dropdown>;
+
+const Template: ComponentStory<typeof Dropdown> = (args: DropdownProps) => <Dropdown {...args} />;
+
+const menu = (
+  <Menu style={{ width: 320 }}>
+    <MenuItem>Dropdown 1</MenuItem>
```

Comment:
`MenuItem` needs to include `key` in the prop, else there will be a warning in the console. can check the `Menu` stories and see how it is done.